In [1]:
import pandas as pd

df=pd.read_csv("data/统计数据.csv",encoding="gbk")
#读取csv：中文通常是GBK编码，用utf-8会报错
df

,地区,地区生产总值(亿元),一般预算收入(亿元),商品房平均销售价格(元/平方米),颗粒物排放量(万吨),每十万人高校在校生数,医疗卫生机构床位数(万张)
0,北京市,41610.9,5714.36,38240,0.41,5397,13.39
1,天津市,16311.3,1846.69,15572,0.90,5428,6.85
2,河北省,42370.4,4056.30,8021,23.65,3070,48.57
3,山西省,25642.6,3453.99,6713,26.63,3388,22.84
4,内蒙古自治区,23158.6,2824.39,6288,99.30,2493,16.77
5,辽宁省,28975.1,2525.07,8315,21.77,3931,32.62
6,吉林省,13070.2,851.00,6955,16.61,4989,17.72
7,黑龙江省,15901.0,1290.66,6152,34.76,3734,26.13
8,上海市,44652.8,7608.00,40302,0.83,3756,16.53
9,江苏省,122875.6,9258.88,12226,9.09,3726,56.30


In [2]:
df["地区"]  #选一列，输出的是列

0          北京市
1          天津市
2          河北省
3          山西省
4       内蒙古自治区
5          辽宁省
6          吉林省
7         黑龙江省
8          上海市
9          江苏省
10         浙江省
11         安徽省
12         福建省
13         江西省
14         山东省
15         河南省
16         湖北省
17         湖南省
18         广东省
19     广西壮族自治区
20         海南省
21         重庆市
22         四川省
23         贵州省
24         云南省
25       西藏自治区
26         陕西省
27         甘肃省
28         青海省
29     宁夏回族自治区
30    新疆维吾尔自治区
Name: 地区, dtype: object

In [3]:
df[["地区"]]  #输出的是表格

,地区
0,北京市
1,天津市
2,河北省
3,山西省
4,内蒙古自治区
5,辽宁省
6,吉林省
7,黑龙江省
8,上海市
9,江苏省


In [4]:
df[["地区","地区生产总值(亿元)"]]##选多列是通常用双层括号

,地区,地区生产总值(亿元)
0,北京市,41610.9
1,天津市,16311.3
2,河北省,42370.4
3,山西省,25642.6
4,内蒙古自治区,23158.6
5,辽宁省,28975.1
6,吉林省,13070.2
7,黑龙江省,15901.0
8,上海市,44652.8
9,江苏省,122875.6


In [5]:
df[df["地区生产总值(亿元)"]>50000]##条件筛选

,地区,地区生产总值(亿元),一般预算收入(亿元),商品房平均销售价格(元/平方米),颗粒物排放量(万吨),每十万人高校在校生数,医疗卫生机构床位数(万张)
9,江苏省,122875.6,9258.88,12226,9.09,3726,56.30
10,浙江省,77715.4,8039.85,18576,6.61,2763,38.17
12,福建省,53109.9,3339.21,10771,7.98,3277,23.24
14,山东省,87435.1,7104.10,8393,19.11,3663,69.36
15,河南省,61345.1,4250.35,6133,6.96,3652,75.22
16,湖北省,53734.9,3281.13,8553,12.70,4110,45.03
18,广东省,129118.6,13260.88,14985,13.40,3129,60.83
22,四川省,56749.8,4880.55,8153,15.19,3127,68.39


In [6]:
df.sort_values("商品房平均销售价格(元/平方米)",ascending=False)  
#排序按商品房平均销售价格(元/平方米)
#False 是从大到小True是从小到大

,地区,地区生产总值(亿元),一般预算收入(亿元),商品房平均销售价格(元/平方米),颗粒物排放量(万吨),每十万人高校在校生数,医疗卫生机构床位数(万张)
8,上海市,44652.8,7608.00,40302,0.83,3756,16.53
0,北京市,41610.9,5714.36,38240,0.41,5397,13.39
10,浙江省,77715.4,8039.85,18576,6.61,2763,38.17
20,海南省,6818.2,832.43,17050,0.75,2999,6.12
1,天津市,16311.3,1846.69,15572,0.90,5428,6.85
18,广东省,129118.6,13260.88,14985,13.40,3129,60.83
9,江苏省,122875.6,9258.88,12226,9.09,3726,56.30
12,福建省,53109.9,3339.21,10771,7.98,3277,23.24
26,陕西省,32772.7,3311.57,9884,21.08,4409,28.96
16,湖北省,53734.9,3281.13,8553,12.70,4110,45.03


In [7]:
gdp = "地区生产总值(亿元)"

print(f"平均值  : {df[gdp].mean():>12,.2f} 亿元")
print(f"中位数  : {df[gdp].median():>12,.2f} 亿元")
print(f"最大值  : {df[gdp].max():>12,.2f} 亿元")
print(f"最小值  : {df[gdp].min():>12,.2f} 亿元")
print(f"标准差  : {df[gdp].std():>12,.2f} 亿元")
print(f"总和    : {df[gdp].sum():>12,.2f} 亿元")

平均值  :    38,821.36 亿元
中位数  :    29,129.00 亿元
最大值  :   129,118.60 亿元
最小值  :     2,132.60 亿元
标准差  :    31,350.73 亿元
总和    : 1,203,462.20 亿元


In [8]:
df["地区生产总值(亿元)"].max() ##只给数字，不知道是那个省

129118.6

In [9]:
i=df["地区生产总值(亿元)"].idxmin()  #返回行号，.idxmax最大值所在行号idxmin同理
df.loc[i,"地区"]  #  .loc[行号,列名] 按行号+列名取值

'西藏自治区'

## 分析结论

**数据**：全国 31 个省份，7 项经济与民生指标

### 1. 经济总量高度集中在少数省份

GDP 超过 5 万亿元的省份有 **8 个**（江苏、浙江、福建、山东、河南、湖北、广东、四川），约占全国的 1/4。

最高的是 **广东省（129118.6 亿元）**，最低的是 **西藏自治区（2132.6 亿元）**，**相差约 60 倍**。

### 2. 平均值明显高于中位数 —— 数据右偏

| 指标 | 数值 |
| --- | --- |
| 平均值 | 38821.36 亿元 |
| 中位数 | 29129.00 亿元 |

**平均值比中位数高出近 1 万亿**，说明少数经济大省把平均水平拉高了。**实际上有一半以上的省份，GDP 低于平均值。**

> **这就是为什么看数据不能只看平均值** —— 中位数才能反映"典型省份"的真实水平。

### 3. 房价最高的不是经济最强的

| 排名 | 省份 | 房价（元/㎡） | GDP（亿元） |
| --- | --- | --- | --- |
| 1 | 上海 | 40302 | 44652.8 |
| 2 | 北京 | 38240 | 41610.9 |
| 3 | 浙江 | 18576 | 77715.4 |

**浙江的 GDP 远高于上海和北京，但房价只有它们的一半左右。**

**结论**：房价不只由经济水平决定，还受**人口密度、土地供应、城市定位、政策调控**等因素影响。

### 4. 值得进一步探索的问题

- 房价与 GDP、人口、高校在校生数之间，谁的相关性更强？
- 颗粒物排放量与经济发展水平是什么关系？
- 能否用这些指标做一个简单的预测模型？（→ 这就进入机器学习范畴了）

---

*分析日期：2026-09-13*
